docs/api/declarations.rst

```diff
@@ -324,6 +335,8 @@ Removing an interface that is provided through the class is not possible:
 classProvides
 -------------
 
+(The `provider` decorator is preferred to this.)
```

Comment:
I think the example here should be moved down to the `provider` section. 

This section should either be removed, or a comment added to say that `classProvides` is deprecated and only works on Python 2; no example is provided.

Reply:
Good idea, thank you, done.